### DISCUSSION TITLE
Does Tailwind work with lit-html?

### DISCUSSION THREAD

**Discussion by SuperstringDev** [tailwindlabs/tailwindcss#11460] [Help] [answered]
I'm thinking that tailwind might be a good fit with lit-html but I'm not sure. I have seen a few articles that propose huge workaround setups just to work with tailwind when using lit-element. In my case I am interested in only lit-html not lit-element.

Hopefully it is straight forward to get going with lit-html and template literals. Any advice is appreciated.

**Comment by wongjn** [comment, marked as answer]
Tailwind should work with `lit-html`. Be sure to [install Tailwind as per your build system (Vite, Webpack, CLI)](https://tailwindcss.com/docs/installation), [configure your `content` file globs](https://tailwindcss.com/docs/content-configuration) and you should be good to go. [Here is an example on Stackblitz with Vite](https://stackblitz.com/edit/vitejs-vite-cxj9gt?file=main.js).

**Reply by SuperstringDev** [reply]
Thanks, that is the perfect combo of tools I'm looking into setting up here!
